dockerfiles/mlrun-api/Dockerfile

```diff
@@ -71,7 +71,7 @@ ENV UVICORN_TIMEOUT_KEEP_ALIVE=${MLRUN_HTTPDB__HTTP_CONNECTION_TIMEOUT_KEEP_ALIV
 COPY ./pipeline-adapters ./pipeline-adapters
 
 RUN uv pip install ./pipeline-adapters/mlrun-pipelines-kfp-common && \
```

Comment:
no need `uv pip install ./pipeline-adapters/mlrun-pipelines-kfp-common` if you have common as dependency of the `mlrun-pipelines-kfp-v1-8` itself, right?

Reply:
This is done because this way we always get the latest version from the source tree, as opposed to having to wait for a release.